package.json

```diff
@@ -39,6 +41,7 @@
     "@rollup/plugin-commonjs": "^17.1.0",
     "@rollup/plugin-json": "^4.1.0",
     "@rollup/plugin-node-resolve": "^11.2.0",
+    "@types/acorn": "^4.0.6",
```

Comment:
Acorn itself has a `*.d.ts` file and the mod date appears more recent, so is this this still necessary?